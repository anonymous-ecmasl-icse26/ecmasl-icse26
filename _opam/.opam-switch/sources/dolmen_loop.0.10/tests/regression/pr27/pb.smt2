(set-logic AUFLIA)
(declare-fun notindomain (Int) Int)
(assert (forall ((i Int))
                (= (notindomain i) (or (< i 0) (< 5 i)) )))
(check-sat)
