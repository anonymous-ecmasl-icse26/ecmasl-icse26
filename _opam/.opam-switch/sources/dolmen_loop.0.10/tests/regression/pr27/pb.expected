File "tests/regression/pr27/pb.smt2", line 4, character 16-57:
4 |                 (= (notindomain i) (or (< i 0) (< 5 i)) )))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error The term: `(i < 0) âˆ¨ (5 < i)` has type `Prop`
      but was expected to be of type `int`
